lib/PuppeteerSharp/Helpers/TaskHelper.cs

```diff
@@ -9,22 +9,33 @@ namespace PuppeteerSharp.Helpers
     /// </summary>
     public static class TaskHelper
     {
+        private static readonly Func<int, Exception> DefaultExceptionFactory =
```

Comment:
How about if we make it TimeSpan by default?
The methods expecting milliseconds can use TimeSpan.FromMilliseconds(int)
And we wouldn't need a `ToTimeoutInt32`

Reply:
True, and I considered it, but `CancellationTokenSource` will convert the `TimeSpan` back to `int` and also throw a `ArgumentOutOfRangeException`.

Reply:
Genius!